apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/injectable/PropertiesHolder.java

```diff
@@ -8,23 +8,29 @@
 import com.apollocurrency.aplwallet.apl.util.annotation.TransactionFee;
 import lombok.extern.slf4j.Slf4j;
 
-import javax.enterprise.inject.Vetoed;
 import java.io.UnsupportedEncodingException;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 import java.util.Properties;
+import javax.enterprise.inject.Vetoed;
 
 @Slf4j
 @Vetoed
 public class PropertiesHolder {
+    private final Properties properties;
 
-    private Properties properties;
-
-    public void init(Properties properties) {
+/**
+ * This is just for mocking in tests. Please do not use it in the real code
+ */    
+    public PropertiesHolder(){
```

Comment:
Maybe better to rewrite tests to not use such dangerous code?

Reply:
Agree, but it is too much of work, let's do it later